FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -153,6 +154,16 @@ void FireRenderContext::resize(unsigned int w, unsigned int h, bool renderView,
 	setResolution(w, h, renderView, glTexture);
 }
 
+int FireRenderContext::GetAOVMaxValue()
```

Comment:
Please make this function as virtual and implement it in TahoeCiontext class. including TahoeContext.h here is not good since TahieContext is child class or FireRenderContext

Reply:
done